_tour/basics.md

```diff
@@ -110,6 +110,14 @@ You can define an anonymous function (i.e. no name) that returns a given integer
 
 On the left of `=>` is a list of parameters. On the right is an expression involving the parameters.
 
+You can give value to the anonymous function as below.
+
+{% scalafiddle %}
+```tut
+println(((x: Int) => x + 1)(5)) //6
```

Comment:
I do not think it is not a common practice to pass arguments to anonymous function diretly.
I suggest a below.

```scala
val func = (x: Int) => x + 1
println(func(5)) // 6
println(func(10)) // 11
```

Reply:
i think it's good to have both example

Reply:
Thank you very much for your comments.
I understood my example is strange.

The example that @exoego suggested has been already exist. Therefore, I just close this PR.

Reply:
Thank you very much for your comments.
I understood my example is strange.

The example that @exoego suggested has already existed. Therefore, I just close this PR.